content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/_cq_dialog/.content.xml

```diff
@@ -38,6 +38,84 @@
+                    name="./youtubeAutoPlay">
+                    <granite:rendercondition
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/simple"
+                        expression="${cqDesign.youtubeAutoPlayEnabled == 'true'}"/>
+                </autoplay>
+                <loop
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Enable Loop"
+                    checked="${cqDesign.youtubteLoopDefaultValue == 'true'}"
+                    uncheckedValue="false"
+                    value="true"
+                    fieldDescription="In the case of a single video specifies whether the player should play the initial video again and again. In the case of a playlist, the player plays the entire playlist and then starts again at the first video."
+                    name="./youtubeLoop">
+                    <granite:rendercondition
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/simple"
+                        expression="${cqDesign.youtubeLoopEnabled == 'true'}"/>
+                </loop>
+                <inline
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Enable Inline Playback (iOS)"
+                    checked="${cqDesign.youtubePlaysInlineDefaultValue == 'true'}"
+                    uncheckedValue="false"
+                    value="true"
+                    fieldDescription="This parameter controls whether videos play inline (on) or fullscreen (off) in an HTML5 player on iOS."
+                    name="./youtubePlaysInline">
+                    <granite:rendercondition
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/simple"
+                        expression="${cqDesign.youtubePlaysInlineEnabled == 'true'}"/>
+                </inline>
+                <rel
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    text="Restrict Related Videos"
+                    checked="${cqDesign.youtubeRelatedVideosDefaultValue == 'true'}"
+                    uncheckedValue="false"
+                    value="true"
+                    fieldDescription="If enabled related videos will come from the same channel as the video that was just played, otherwise from any channel."
```

Comment:
It actually works the other way around (see other comments).

Reply:
Fixed in https://github.com/adobe/aem-core-wcm-components/pull/1301/commits/86df8b09bbad467797d30dbc05f87dc90d360de9.